tasks/main.yml

```diff
@@ -15,6 +15,21 @@
   include: "DebianInstall.yml"
   when: ansible_os_family == "Debian"
 
+- name: "Determine region of the instance"
```

Comment:
I haven't been able to reproduce this issue. If you are still running into issues, can you provide the version of Ansible, and the OS (the AMI if possible). I'm guessing this was an issue with the ec2-facts module, which has been fixed.